tracekit.js

```diff
@@ -149,27 +160,31 @@ TraceKit.report = (function reportModuleWrapper() {
      * @memberof TraceKit.report
      * @throws An exception if an error occurs while calling an handler.
      */
-    function notifyHandlers(stack, isWindowError, error) {
+    function notifyHandlers(stack, isWindowError, aArguments) {
         var exception = null;
         if (isWindowError && !TraceKit.collectWindowErrors) {
           return;
         }
         for (var i in handlers) {
-            if (_has(handlers, i)) {
+        if (_has(handlers, i) && handlers[i][1] === TraceKit.windowPointer) {
                 try {
-                    handlers[i](stack, isWindowError, error);
+                  var errorObj=(aArguments.length > 4) ? aArguments[4] : null;
+                  handlers[i][0](stack, isWindowError, errorObj);
```

Comment:
This is kind of confusing, so we call a method with stack, isWindowErorr and an array with 4 items? We seem to only be using the error object, so it might be better to revert this signature to just be error object.  Previous thought: Maybe we need to add a comment stating that these arguments are actually: `message, url, lineNo, columnNo, errorObj` (if we were on es6, restructuring would make this so much more readable but we can't use that :(`,